### PR TITLE
reformatted JavaConcepts

### PR DESCRIPTION
**Pull request by JimmyGan437**
I don't see any tests that currently uses JavaConcepts (besides for prettyPrinting, but they already uses a different copy of the file).

### CODE REVIEW

**Comment by jlerbsc** [comment]
Can you check that the project can be build before pushing a PR? Thank you.

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -14,5 +14,5 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.5/apache-maven-3.9.5-bin.zip
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```
Comment (jlerbsc) [inline .mvn/wrapper/maven-wrapper.properties]: Why did you downgrade this version?

Comment (JimmyGan437) [reply .mvn/wrapper/maven-wrapper.properties]: Honestly, I don't really know how that happend. The changes have been reverted. I apologies for the trouble

**Comment by jlerbsc** [comment]
Why did you remove all your changes? I'm going to close this PR because it introduces changes in one commit and removes them in another. Can you make a PR that is easier to read by checking the code you commit?

**Pull request closed without merging**